CHANGELOG.rst

```diff
@@ -5,6 +5,70 @@ CHANGE THIS IN changelogs/config.yaml! Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Minor Changes
+-------------
+
+- Add a playbook to deploy a simple flask web app into high availability architecture (https://github.com/redhat-cop/cloud.aws_ops/pull/97).
```

Comment:
I guess some new roles are not shown in the changelog. I don't know if there's a way to automatically add then at changelog generation, but if not, we should manually create a changelog for them.

Reply:
done